tests/multi/multi.act

```diff
@@ -0,0 +1,24 @@
+behaviour remote of B
+interface set_remote(uint y)
+
+iff
+   CALLVALUE == 0
+
+// we can specify how a call to a function of B
+// updates the storage of anothere by labelling the storage header
```

Comment:
Typo `another`

Reply:
Wouldn't this idea of specifying how `a`'s storage in `B`'s spec get too convoluted at some point? IMO it would be modular so that `B`'s spec only knows that a certain function from `a` is called. By analyzing that function in `A` we could infer what happens to `B.a`.

Reply:
Hmm, I don't think I like that sort of indirection in specs. It would make it harder to see what's actually going on. And it would only work for contracts which has specs already. Yes, contracts that touch a lot of other stuff would have large specs, but that does tell you something about their complexity

Reply:
Thinking about this more, I think I can express this "indirection" concern in a more precise way. Since specs specify state transitions with pre and post state, composing several state transitions to make a larger one would require a careful ordering of them. 

It is not sufficient to say that `B` calls `a`, we need to say at what point this happens.

If I have a state transition `f` that calls `g` (for which we have a spec) somewhere in the middle, I would need to break `f` down into `f0`, which specifies how state changes prior to `g` being called and `f1` which specifies how the state changes afterwards. Then I could express `f` as `f0 ~> g ~> f1`. 

This composition is definitely useful, but I don't think that `behaviours` would be the right place for it. Behaviours, in my view, should specify the state transition as a pre-post pair. Combining state transitions can happen in a property section, or in other backends

Reply:
I guess "inlining" the state updates would remove the need for this split into `f0` and `f1`.

Reply:
How would that look? Currently state updates aren't specified sequentially so I don't see where it could be squeezed in